app/src/main/java/com/glia/exampleapp/Utils.java

```diff
@@ -211,7 +212,7 @@ public static Integer getTypefaceFromPrefs(SharedPreferences sharedPreferences,
         String delius = resources.getString(R.string.font_delius_value);
         String expletus = resources.getString(R.string.font_expletus_value);
         String tangerine = resources.getString(R.string.font_tangerine_value);
-        String defaultPrefValue = resources.getString(R.string.settings_value_default);
+        String defaultPrefValue = resources.getString(R.string.settings_value_default); // todo Wrong string resource is used. should be R.string.default_value
```

Comment:
NIT: I think it would be better if all TODO notes would have some related Jira tickets attached. Otherwise, these improvements/changes will not get the required attention in the future.